src/main/java/de/tum/in/www1/artemis/service/connectors/LtiService.java

```diff
@@ -370,21 +317,22 @@ private void saveLtiUserId(User user, String ltiUserIdString) {
     }
 
     /**
-     * Checks if a LTI request is correctly signed via OAuth with the secret
+     * Checks if an LTI request is correctly signed via OAuth with the secret
```

Comment:
```suggestion
     * Checks if a LTI request is correctly signed via OAuth with the secret
```

Reply:
Interestingly I also had it like this originally but IntelliJ (correctly) suggests to use "an" since the capital "L" sounds like a vowel. ["Use a or an before an acronym or abbreviation depending on how it is pronounced instead of how it is written. If it starts with a consonant sound when read aloud, use a; if it starts with a vowel sound, use an."](https://editorsmanual.com/articles/indefinite-article-a-an-with-abbreviations/)